R/plot-histogram.R

```diff
@@ -37,6 +41,7 @@ plotHistogram <- function(data = NULL,
                           metaData = NULL,
                           x = NULL,
                           dataMapping = NULL,
+                          frequency = NULL,
```

Comment:
same

Reply:
In plot functions, I had made the choice of leaving `NULL` as the default value for optional arguments.
So that they won't overwrite the values created by `dataMapping` and `plotConfiguration`.
This aims at making it easier to centralize and modify the default `frequency`, `bins`, `distribution`, etc. as it is managed only by the `HistogramDataMapping` object.
I can change it if it makes it clearer for the users.